.github/workflows/rust-static-lib.yml

```diff
@@ -69,5 +69,5 @@ jobs:
         run: |
           export WASMEDGE_DIR="$(pwd)/../../"
           export WASMEDGE_BUILD_DIR="$(pwd)/../../build"
-          cargo test --workspace --features static --locked
-          cargo test --examples --features static --locked
+          cargo test --workspace --features static --locked -- --test-threads=1
```

Comment:
Why do we need to add this limit? Is there any new requirement changed in this PR?

Reply:
In this PR we added a new test case `test_crazy_function_creation`. This test function is used to verify the global static variable `HOST_FUNCS` which is used to store the host functions. However, other test cases also access the global variable, therefore, use `-- --test-threads=1` to avoid the case of other test cases modifying `HOST_FUNCS` while `test_crazy_function_creation` is running. In short, the option turns off the execution of all test cases in parallel.